server/src/lib/score-import/framework/score-importing/derivers.ts

```diff
@@ -77,10 +77,7 @@ export function CreateEnumIndexes<GPT extends GPTString>(gpt: GPT, metrics: any,
 		indexes[key] = index;
 	}
 
-	for (const [key, conf] of [
-		...Object.entries(gptConfig.providedMetrics),
-		...Object.entries(gptConfig.derivedMetrics),
-	]) {
+	for (const [key, conf] of [...Object.entries(gptConfig.optionalMetrics)]) {
```

Comment:
jesus this is a pretty rough bug. will need to merge this and then maybe run some rederivations on the database....